common/common_target.mk

```diff
@@ -50,6 +50,8 @@ else
 	UNAME:=$(shell uname -s)
 	ifeq ($(UNAME), Linux)
 		OS:=linux
+	else
```

Comment:
agreed. The Windows side is really Cygwin / Babun, so it needs some work, but yeah, uname exists on OSX